README.md

```diff
@@ -12,10 +12,12 @@ template for creating websites for workshops.
     [automatically published as a website by GitHub][github-project-pages].
 
 3.  Once you are done,
-    please **send your repository's URL to the [Software Carpentry administrator][contact]**.
+    please access [AMY's self-organized workshop form][self-organized-workshop-form],
```

Comment:
"please fill in [this self-organized workshop form][self-organized-workshop-form] and the administrator will contact you if we need any extra information."